### PR TITLE
Apply sponsors list screen design

### LINKED ISSUE #179: apply  sponsors list screen design
**Idea Description**
apply  sponsors list screen design

**Reference images and links**
coming soon

### PR DESCRIPTION
**Pull request by ho2ri2s**
## Issue
- close #179 

## Overview (Required)
- Apply sponsors list data to UI with Compose.
- Not implement Test.

## Screenshot
Before | After
:--: | :--:
 | 

## Movie

https://user-images.githubusercontent.com/43269230/191479009-e4773e80-0512-487b-a492-d2180f6e6507.mp4

### CODE REVIEW

**Comment by ho2ri2s** [comment]
Once the screen is implemented 🙏 
Since the test is not yet supported, could you please create a separate Issue?

**Comment by takahirom** [comment]
It's OK!
Can you run `./gradlew composeLint` ?

File: feature/sponsors/src/main/java/io/github/droidkaigi/confsched2022/feature/sponsors/Sponsors.kt [hunk 1]
```diff
@@ -54,18 +81,129 @@ fun Sponsors(
...
+private fun SponsorTitle(titleItem: SponsorItem.Title) {
+    Text(
+        modifier = Modifier.padding(vertical = 8.dp),
+        text = stringResource(resource = titleItem.value),
+        color = MaterialTheme.colorScheme.onSurface,
+        style = MaterialTheme.typography.titleLarge
+    )
+}
+
+private fun LazyGridScope.sponsorsList(
```
Comment (takahirom) [inline feature/sponsors/src/main/java/io/github/droidkaigi/confsched2022/feature/sponsors/Sponsors.kt:151]: 🆒

**Comment by ho2ri2s** [comment]
Fixed lint related issues.
Please check it when you have time.

**Review by takahirom — APPROVED** [review]
Looks great! Thanks!

**Pull request merged**